src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -91,9 +99,9 @@
    public void initializeUI(UIBuilder builder) throws Exception
    {
       UIContext context = builder.getUIContext();
+      Mission mission = (Mission) context.getAttributeMap().get(Mission.class);
```

Comment:
Are you sure? Look at the original code I replaced in line 108. That was why I moved it outside the lambda to the top because I thought it was duplicated code ^^
(Btw, things seem to work, but perhaps it's only in certain situations where my code will fail?)